rocon_python_utils/src/rocon_python_utils/ros/fileio.py

```diff
@@ -42,4 +42,21 @@ def get_rocon_home():
     rocon_home = os.path.join(rospkg.get_ros_home(), 'rocon')
     if not os.path.isdir(rocon_home):
         os.makedirs(rocon_home)
-    return os.path.join(rospkg.get_ros_home(), 'rocon')
\ No newline at end of file
+    return os.path.join(rospkg.get_ros_home(), 'rocon')
+
+def is_validation_file(file_path):
+  '''
+  Investigate whether file is validated or not as checking file existed in path and file size
+
+  :param file_path: full file path to check
+  :type file_path: string
+  
+  :return: validation of file
+  :rtype: bool
+  '''
+  is_validated = False
+  if not os.path.isfile(file_path) or os.stat(file_path).st_size <= 0:
+      is_validated = True
```

Comment:
> Shouldn't this be False?
It it wrong if statement. I will fix it.

> Also what is the purpose of checking for a size < 0....when does this occur (curious)?
The reason of checking file size is to prevent usage of files do not have any content.